components/src/core/components/CardTable/Decorator/DefaultCardWeb.vue

```diff
@@ -49,6 +49,7 @@ export default defineComponent({
           'oxd-table-card': true,
           [`oxd-row-highlight--${highlightObject.type}`]:
             highlightObject.rowIndexes.indexOf(index) > -1,
+          'oxd-row-selected': this.tableProps.selected.indexOf(index) > -1,
```

Comment:
The code above this new like also does a similar thing.

[`oxd-row-highlight--${highlightObject.type}`]:
            highlightObject.rowIndexes.indexOf(index) > -1,

Why do we need to add a new class as 'oxd-row-selected' here? Can't it be satisfied from the 'oxd-row-highlight--' classes. I think the intention is same.

Reply:
I just wanted to separate the row select background colour when marked rows by checking and the highlight colour for special events such as "newly added"

Reply:
Used the same highlight option